files/default/neutron-ha-tool.py

```diff
@@ -22,6 +22,7 @@
 import argparse
 import random
 import time
+import retrying
 from logging.handlers import SysLogHandler
 from collections import OrderedDict
```

Comment:
Taken care of by https://build.suse.de/package/view_file/Devel:Cloud:5:Staging/openstack-neutron/neutron-ha-tool_python26-ordereddict.patch?expand=1